packages/arb-token-bridge-ui/src/components/common/UnsupportedNetworkContent.tsx

```diff
@@ -0,0 +1,47 @@
+import { useWallet } from '@arbitrum/use-wallet'
+import { Web3Provider } from '@ethersproject/providers'
+import { ChainId, getNetworkName } from '../../util/networks'
+import { supportedNetworks, switchChain } from '../../util/NetworkUtils'
+import { Button } from './Button'
+
+export const UnsupportedNetworkContent = () => {
```

Comment:
Being nitpicky here, but maybe name this `MainNetworkNotSupported` so it matches with `HeaderNetworkNotSupported`?